examples/Azure_IoT_Central_ESP32_AzureIoTKit/AzureIoT.cpp

```diff
@@ -0,0 +1,1258 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I probably won't go through this whole file since it seems this is very similar (if not identical?) to the other one. I trust you to take the feedback from there and apply what's relevant here :)

Reply:
going to assume the difference is in Azure_IoT_PnP_Template.cpp

Reply:
You are both correct. I replicated all the changed between the 2 samples. These files are common between the samples.